### PR TITLE
[DPB]Removing default admin status initialization in DPB flow while loading minigraph

### PR DESCRIPTION
**Pull request by dgsudharsan**
#### Why I did it
To Fix https://github.com/Azure/sonic-buildimage/issues/8697 . The config load_minigraph initializes 'admin_status' to up when platform.json has DPB configs. This doesn't happen when using port_config.ini
The update minigraph has logic to initialize only the ports whose neighbors are defined or those belonging to portchannel
However, a change was introduced to have default admin status to be 'up' in portconfig.py when the minigraph was using platform.json

This will lead to sanity check failure in sonic-mgmt and thus no test cases could be run

#### How I did it
Fixed the platform.json flow not to initialize admin_status by default. Fixed UT to ensure that different cases are handled.

#### How to verify it
With dpb enabled perform config load_minigraph

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [x] 202106

### CODE REVIEW

**Comment by dgsudharsan** [comment]
@zhenggen-xu Please review

**Review by liat-grozovik — DISMISSED** [review]

**Comment by liat-grozovik** [comment]
@lguohan and @yxieca  could you please help to review or assign someone?
This is blocking issue for running sonic mgmt tests on 202106/master

File: src/sonic-config-engine/tests/sample_output/platform_output.json [hunk 1]
```diff
@@ -59,8 +55,8 @@
     "index": "2",
     "lanes": "6,7",
     "description": "Eth2/2",
-    "admin_status": "up",
     "mtu": "9100",
+    "admin_status": "up",
```
[the remark this answers is not in the document]
Comment (dgsudharsan) [reply src/sonic-config-engine/tests/sample_output/platform_output.json]: Modified it back. It was not needed.

**Review by zhenggen-xu — APPROVED** [review]

**Comment by dgsudharsan** [comment]
@lguohan Can you please review and merge? This fix is crucial for running regression on 202106

**Pull request merged**